presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueriesGlueMetastore.java

```diff
@@ -0,0 +1,446 @@
+                .put("hive.allow-drop-column", "true")
+                .build();
+
+        queryRunner.createCatalog("test_glue", "glue", glueProperties);
+    }
+
+    @Override
+    protected Session getSession()
+    {
+        return Session.builder(super.getSession())
+                // This option is required to avoid the error:
+                // "Unknown plan node type com.facebook.presto.sql.planner.plan.TableWriterMergeNode".
+                // TableWriterMergeNode is not supported by Velox yet.
+                .setSystemProperty("table_writer_merge_operator_enabled", "false")
+                .build();
+    }
+
+    private static String getRandomName(String prefix)
+    {
+        String randomName = UUID.randomUUID().toString().toLowerCase(ENGLISH).replace("-", "");
+        return TEMPORARY_PREFIX + prefix + randomName;
+    }
+
+    /**
+     * Returns the fully-qualified schema name.
+     */
+    private static String tempSchema(String prefix)
+    {
+        return CATALOG_NAME + "." + getRandomName(prefix);
+    }
+
+    /**
+     * Returns the fully-qualified table name.
+     */
+    private static String tempTable(String schemaName, String prefix)
+    {
+        return schemaName + "." + getRandomName(prefix);
+    }
+
+    @BeforeClass
+    public void setUp() throws IOException
+    {
+        tempWarehouseDir = Files.createTempDir();
+        setUpGlueCatalog(getQueryRunner(), tempWarehouseDir);
+        setUpGlueCatalog((QueryRunner) getExpectedQueryRunner(), tempWarehouseDir);
+
+        schemaName = tempSchema("test_schema");
+        getQueryRunner().execute("CREATE SCHEMA " + schemaName);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void tearDown() throws IOException
+    {
+        deleteRecursively(tempWarehouseDir.toPath(), ALLOW_INSECURE);
+        if (schemaName != null) {
+            getQueryRunner().execute("DROP SCHEMA IF EXISTS " + schemaName);
+        }
+    }
+
+    @Test
+    public void testShowCatalogs()
+    {
+        // Glue catalog should be configured correctly.
+        MaterializedResult res = getQueryRunner().execute("show catalogs like '%" + CATALOG_NAME + "%'");
+        assertEquals(res.getMaterializedRows().size(), 1);
+        for (MaterializedRow row : res.getMaterializedRows()) {
+            assertEquals(row.getField(0), CATALOG_NAME);
+        }
+    }
+
+    @Test
+    public void testCreateDropSchema() throws Exception
+    {
+        String schemaName = tempSchema("default");
+        getQueryRunner().execute("CREATE SCHEMA " + schemaName);
+        getQueryRunner().execute("DROP SCHEMA " + schemaName);
+    }
+
+    @Test
+    public void testCreateInsertDropTable() throws Exception
+    {
+        String tableName = tempTable(schemaName, "test_table");
+        Session session = getSession();
+
+        try {
+            getQueryRunner().execute(session, "CREATE TABLE " + tableName + "(a int, b int) WITH (format = 'DWRF')");
```

Comment:
DWRF format is only used by Meta and very small number of other users internally. Most people use Parquet and ORC. Let's either 
1) use Parquet here, or 
2) you can extract a private method that takes the file format parameter and test all 3 formats.

Reply:
Actually, HiveExternalWorkerQueryRunner uses DWRF. 

When switching to Parquet, I encountered `VeloxRuntimeError: it != readerFactories().end() ReaderFactory is not registered for format parquet Split [file file:/private/var/folders/jh/xnfw3kr91kv37pmkhw4mnr7w0000gn/T/1680030209091-0/tmp_presto_glue_test_test_schemab05aef72e3fa4f438e09490430e8c6e1/tmp_presto_glue_test_test_tablef5b0153a07084f0ca247b79a2d0dae35/20230328_190335_00001_9t2qf.1.0.3_0_e9f4720e-d24d-4b34-a4f5-9ae708c33b61 0 - 2966] Task 20230328_190342_00003_9t2qf.1.0.0` error that is why I decided to use DWRF which appears to be registered by default.

Reply:
I asked how to mitigate the issue on the Slack channel: https://ahana-io.slack.com/archives/C02AK36GN22/p1680031114236749

Reply:
Based on the discussion, I am going to use DWRF, it seems Parquet requires special environment variable during build which would mean modifying GitHub CI. Maybe it is better to follow up on that separately once the PR is merged.

Reply:
@isadikov 
I am surprised that there is not any Parquet related native e2e tests so far. For now, I think it'll be best to add `-DPRESTO_ENABLE_PARQUET=ON` in your CI change and test all of Parquet, DWRF and ORC here. I'm suggesting this because this DOES involve Java code change, and we want to test Parquet anyways. As you can see, it may expose problems that a specific reader factory may not be registered properly. In the near future we do need to add more Parquet coverages in "test native" job and that can be done separately.